contracts/test/v0.8/foundry/vrf/VRFV2PlusWrapper.t.sol

```diff
@@ -135,13 +153,46 @@ contract VRFV2PlusWrapperTest is BaseTest {
   // VRFV2PlusWrapperConsumerBase events
   event LinkTokenSet(address link);
 
+  // SubscriptionAPI events
+  event SubscriptionConsumerAdded(uint256 indexed subId, address consumer);
+
   function testVRFV2PlusWrapper_ZeroAddress() public {
     vm.expectRevert(VRFConsumerBaseV2Plus.ZeroAddress.selector);
-    new VRFV2PlusWrapper(address(0), address(0), address(0));
+    new VRFV2PlusWrapper(address(s_linkToken), address(0), address(0), uint256(0));
   }
 
-  function testSetLinkNativeFeed() public {
-    VRFV2PlusWrapper wrapper = new VRFV2PlusWrapper(address(s_linkToken), address(0), address(s_testCoordinator));
+  function testVRFV2PlusWrapperWithExistingSubscriptionId() public {
+    // second wrapper contract will simply add itself to the same subscription
+    VRFV2PlusWrapper nextWrapper = new VRFV2PlusWrapper(
+      address(s_linkToken),
+      address(0),
+      address(s_testCoordinator),
+      s_wrapperSubscriptionId
+    );
+    assertEq(s_wrapperSubscriptionId, nextWrapper.SUBSCRIPTION_ID());
+
+    vm.expectEmit(true, false, false, true);
+    emit SubscriptionConsumerAdded(uint256(s_wrapperSubscriptionId), address(nextWrapper));
+    s_testCoordinator.addConsumer(uint256(s_wrapperSubscriptionId), address(nextWrapper));
```

Comment:
These lines could be removed, as they are not relevant to VRFV2PlusWrapper functionality testing anymore and have to be externally handled. Having them here, may only impact readability of these tests

Reply:
Sure, they will be removed